tests/test_jupyter_book/test_toc.tex

```diff
@@ -0,0 +1,146 @@
+\begin{document}
+
+\pagestyle{empty}
+\sphinxmaketitle
+\pagestyle{plain}
+\sphinxtableofcontents
+\pagestyle{normal}
+\phantomsection\label{\detokenize{intro::doc}}
+
+
+\sphinxAtStartPar
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis vel nunc vestibulum, pretium urna ac, hendrerit diam. Nulla ut mi sit amet dui placerat tempor at quis magna. Sed porttitor blandit consequat. Fusce efficitur mi eget pharetra cursus. Sed vel blandit dui, et pharetra velit. Ut eget erat rutrum, maximus felis ullamcorper, congue urna. Ut pellentesque, orci vel consectetur aliquam, velit elit vehicula libero, eu blandit tellus risus rhoncus massa. Curabitur id libero ut sapien aliquet dapibus nec et dolor. Pellentesque tincidunt iaculis ipsum vitae pretium. Quisque et egestas orci, sed dignissim sem. Vestibulum sed commodo dui. Nulla rutrum vulputate nisl nec interdum. Cras pharetra quam et tellus convallis, ut hendrerit mi mollis. Integer finibus tellus porttitor tellus bibendum, a auctor purus dapibus. Etiam mauris est, blandit ut dapibus eget, rutrum et dolor.
+
+\begin{DUlineblock}{0em}
+\item[] \sphinxstylestrong{\Large This is H2}
+\end{DUlineblock}
+
+\sphinxAtStartPar
+Etiam interdum tempor augue at volutpat. Nulla sit amet volutpat elit. In vehicula dapibus velit, a placerat neque blandit et. Etiam non mollis nibh. Mauris tortor purus, semper vel libero at, faucibus mattis enim. Sed elit mi, vestibulum sit amet porttitor in, elementum a sem. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec gravida faucibus enim, at venenatis ante. Curabitur viverra neque vel nulla condimentum ultrices quis a ipsum.
+\begin{itemize}
```

Comment:
The current jupyter-book creates this weird list if a `tableofcontents` directive is used.
But this is no longer there when moving to sphinx-external-toc, which I believe is correct.
I've already noted this in the jupyter-book PR: https://github.com/executablebooks/jupyter-book/pull/1293#discussion_r615433968

Reply:
@chrisjsewell will add `tableofcontents` directive logic here, after this PR gets merged.

Reply:
but I don't understand why this list should be there in the first place? Is this something you want?

Reply:
@chrisjsewell because `tableofcontents` directive was transformed into a list for latex. using the `toctree` directive directly in latex(like we do for HTML) includes the files themselves instead of just returning a list of file titles.

Reply:
I tried using `\tableofcontents` command, or packages like [minitoc](https://www.ctan.org/pkg/minitoc#:~:text=The%20minitoc%20package%20allows%20you,every%20chapter%2C%20part%20or%20section.&text=If%20the%20type%20of%20document,level%20secttocs%2C%20sectlofs%20and%20sectlots.) , but all of them had issues. For example, they specified the page numbers as well of entries, something like:
<img width="572" alt="Screen Shot 2021-04-23 at 2 17 51 pm" src="https://user-images.githubusercontent.com/6542997/115817736-c67de880-a43e-11eb-8f7c-6dd7087f9b80.png">
 
But, we wanted a simpler version of it(like that for in HTML), only having the title names, like:

<img width="527" alt="Screen Shot 2021-04-23 at 2 20 59 pm" src="https://user-images.githubusercontent.com/6542997/115817964-21afdb00-a43f-11eb-8b3b-9da1bc5e24a2.png">

 and directly using `toctree` directive did not help(for the reason mentioned in the comment before), so created a bullet list by myself.

Reply:
@AakashGfude I do like the simplicity of these `contents` listings. Perhaps we should keep this as is and iterate to see if we can find a more `LaTeX` ecosystem based approach to solving this problem.